src/mococrw/symmetric_crypto.h

```diff
@@ -56,6 +56,12 @@ enum class SymmetricCipherPadding
     PKCS
 };
 
+enum class SymmetricCipherOperation
```

Comment:
Since this now part of the public interface, please add documentation.

Reply:
Done.
It's private again.